packages/secure_storage/amplify_secure_storage/Makefile

```diff
@@ -1,4 +1,21 @@
 .PHONY: pigeons
-pigeons:
-	flutter pub run pigeon --input pigeons/messages.dart
-	flutter format --fix lib/src/messages.g.dart
+.PHONY: pigeons_android
+.PHONY: pigeons_ios
+.PHONY: pigeons_macos
+
+pigeons: pigeons_android pigeons_ios pigeons_macos
+
+pigeons_android:
+	flutter pub run pigeon --input pigeons/messages.android.dart
+	flutter format --fix lib/src/messages.android.g.dart
+
+pigeons_ios:
+	flutter pub run pigeon --input pigeons/messages.ios.dart
+	flutter format --fix lib/src/messages.cupertino.g.dart
+
+pigeons_macos:
+	flutter pub run pigeon --input pigeons/messages.macos.dart
+	flutter format --fix lib/src/messages.cupertino.g.dart
+	# TODO(Jordan-Nelson): Remove when swift codegen in pigeon is no longer experimental.
+	# See: https://github.com/flutter/flutter/issues/73738#issuecomment-903927725
+	sed -i '' -e 's/\<Flutter\/Flutter\.h\>/\<FlutterMacOS\/FlutterMacOS\.h\>/' macos/Classes/Messages.m
```

Comment:
*Note:* Pigeon doesn't officially support using objective-c for MacOS because flutter doesn't officially support using it for MacOS plugins. However, it works just fine with this one import update. Pigeon recently added support for swift codegen for MacOS, but swift codgen in pigeon is considered experimental.

I think it makes sense to stick with objective-c until the swift code generator is no longer "experimental". At that point we can switch iOS and MacOS over to using swift.